pkg/apis/hco/v1beta1/hyperconverged_types.go

```diff
@@ -46,7 +46,7 @@ type HyperConvergedSpec struct {
 
 	// featureGates is a map of feature gate flags. Setting a flag to `true` will enable
 	// the feature. Setting `false` or removing the feature gate, disables the feature.
-	// +kubebuilder:default={"withHostPassthroughCPU": false, "sriovLiveMigration": true, "enableCommonBootImageImport": true}
+	// +kubebuilder:default={"withHostPassthroughCPU": false, "sriovLiveMigration": true, "enableCommonBootImageImport": false}
```

Comment:
how is this related to kubevirt log verbosity?

Reply:
It's auto-generated by `make generated`... not sure why.. =/

Reply:
No idea how that happened.. fixing

Reply:
Not sure what was going on. it's fixed now.